src/engine/engine.ts

```diff
@@ -51,7 +61,7 @@ export function getOptionsString(options: Schema) {
   return (
     Object.keys(options)
       // Get only options with value
-      .filter(optKey => !!options[optKey])
+      .filter(optKey => !!options[optKey] && optKey !== 'packageVersion')
```

Comment:
This is braking the [Single Responsibility Use Principle](https://medium.com/@severinperez/writing-flexible-code-with-the-single-responsibility-principle-b71c4f3f883f). This function doesn't be aware about what our project needs, that filter needs to be done outside the function

Reply:
I just delete `packgeVersion` from the options after using it  
and I do think there should be a function called `prepareOptionsForNpmCommand` to filter out the options that may cause a malfunction when passing to the command of `npm publish`